crates/wallets/Cargo.toml

```diff
@@ -45,6 +45,7 @@ rpassword = "7"
 serde.workspace = true
 thiserror.workspace = true
 tracing.workspace = true
+eth-keystore = "0.5.0"
```

Comment:
I would like to avoid pulling in this dependency here

I think we could use it from Alloy here:

https://github.com/alloy-rs/alloy/blob/fa01c2ab9174abcd9521624ef341fcfea3d08356/crates/signer-local/src/error.rs#L31-L34

Reply:
Yes, I'm already using it, but I need to specifically catch the `eth_keystore::KeystoreError::MacMismatch` error, right?

```rust
alloy_signer_local::LocalSignerError::EthKeystoreError(
  eth_keystore::KeystoreError::MacMismatch,
)
```

Reply:
Ah yeah.. OK